utils/src/treev2/quickPickWizard/common/PickFilter.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+
+export interface PickFilter<TPick = vscode.TreeItem> {
+    /**
+     * Filters for nodes that match the final target.
+     * @param node The node to apply the filter to
+     */
+    isDirectPick(node: TPick): boolean;
```

Comment:
I wasn't clear what exactly these filters meant by their name alone. The descriptions are perfect, but what if we named them something more like `isFinalPick` and `isAncestorPick`? 

I also like the use of ancestor because it implies that is has to be a parent node.

Reply:
I agree. I like `isFinalPick`, and `isAncestorPick` (I thought about `isIntermediatePick`, but I like `isAncestorPick` better).